qa/backwards/shared/src/test/java/org/elasticsearch/bwcompat/BasicBackwardsCompatibilityIT.java

```diff
@@ -713,4 +715,55 @@ private static String indexOrAlias() {
     private void createIndexWithAlias() {
         assertAcked(prepareCreate("test").addAlias(new Alias("alias")));
     }
+
+    public void testSimpleFunctionScoreParsingWorks() throws IOException, ExecutionException, InterruptedException {
```

Comment:
I'm not a fan of growing this class so much. I'm happy to leave the tests in the class they were in.